match/cameron_so.py

```diff
@@ -2,6 +2,29 @@
 from datamatch import JaroWinklerSimilarity, ThresholdMatcher, ColumnsIndex
 import deba
 from lib.post import load_for_agency
+from lib.clean import canonicalize_officers
+
+
+def deduplicate_cprr14(cprr):
+    df = cprr[["uid", "first_name", "last_name"]]
+    df = df.drop_duplicates(subset=["uid"]).set_index("uid")
+    df.loc[:, "fc"] = df.first_name.fillna("").map(lambda x: x[:1])
+    matcher = ThresholdMatcher(
+        ColumnsIndex("fc"),
+        {
+            "first_name": JaroWinklerSimilarity(),
+            "last_name": JaroWinklerSimilarity(),
+        },
+        df,
+    )
+    decision = 1
```

Comment:
Deduplicate with threshold of `1` feels like a no-op to me. Unless I'm missing something.

Reply:
Thanks.